packages/flutter/lib/src/material/app.dart

```diff
@@ -166,6 +169,17 @@ class MaterialApp extends StatefulWidget {
   /// The colors to use for the application's widgets.
   final ThemeData theme;
 
+  /// The colors to use for the application's widgets when the host platform has
+  /// specifically requested a dark UI.
+  ///
+  /// For example, starting with Android Pie, turning on battery saver causes
```

Comment:
You can use a stronger tone here. e.g. this is only implemented on Android and only on Android Pie (I thought you said Q) and only when androidmanifest says X and your android view has attribute Y.

Reply:
@xster I provided a present-day example, but I avoided using language that limits the example because it is a near certainty that this behavior will be available in new contexts and across multiple platforms.  If we claim that it is only for battery saver on Android Pie then we're just creating a future task to reword that, right? (assuming someone remembers to reword it)